src/RoslynAnalyzers/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.cs

```diff
@@ -4,7 +4,7 @@
 
 using System;
 
-namespace Microsoft.CodeAnalysis.Analyzers
+namespace Microsoft.CodeAnalysis
```

Comment:
Needed as RoslynAnalyzers adds methods to the 'Resources' types by adding partial parts.  Because we are pulling in Resources from a shared location, it only works if we are consistent with using MS.CA as our namespace for resource locations consistently everywhere.